projects/experimental/directives/button-group/button-group.style.less

```diff
@@ -0,0 +1,43 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroup] {
+    display: flex;
+    border-radius: 1.5rem;
+
+    > button {
+        .button-base();
+
+        flex-grow: 1;
```

Comment:
Need transitions. I see in specs :active state has --tui-clear background. Let's also put link color as default in here. And it's missing cursor: pointer

Reply:
fixed